src/components/Bookings.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from "react";
+import "./Bookings.css";
+import Search from "./Search.js";
+import SearchResults from "./SearchResults.js";
+import SearchButton from "./SearchButton.js";
```

Comment:
This import isn't used so should be removed otherwise you will get linting warnings